main.cpp

```diff
@@ -183,105 +298,263 @@ void Vinyl::moveToneArm(bool is12Inch)
 {
     float distanceToMoveArm;
     distanceToMoveArm = is12Inch ? 6.0f : 3.5f; 
+    std::cout << "Move tone arm " << distanceToMoveArm << "inches" << std::endl;
+}
+
+double Vinyl::bringJoy(double initNumberOfGoodTracks, double initListenerMood)
+{
+    double quality = initNumberOfGoodTracks * initListenerMood;
+    return quality;
+}
+
+double Vinyl::accrueValue(double initNumberOfGoodTracks, double playsOnYoutube)
+{
+    double valueScale = 0.00005;
+    double value = initNumberOfGoodTracks * playsOnYoutube;
+    return value * valueScale;
 }
 
 struct Oscillator
 {
-    double sawLevel = 0.0;
-    double squLevel = 1.0;
-    double tuning = 0.0;
-    bool subOscOn = false;
-    int octave = 0; 
+    Oscillator();
+    double sawLevel;
+    double squLevel;
+    double tuning;
+    double subOscOn;
+    int octave;
 
     double osc1FM(double osc1SawSignal, double osc1SquSignal);
-    double subOscSignal(bool subOscOn);
+    double subOscSignal(bool isSubOscOn);
     int octaveLED(int octave, bool shiftButtonPressed);
```

Comment:
These function names do not describe an action or verb.  
You wrote a noun for the function name.

- [ ] revise